src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -344,42 +345,56 @@ internal void StartProcess(string workerId, Process process)
                     else if (e.Data.IndexOf("error", StringComparison.OrdinalIgnoreCase) > -1)
                     {
                         _logger.LogError(e.Data);
+                        processExitMessage += e.Data;
                     }
```

Comment:
We need to control how much of stderr is buffered here and how much of it goes to system logs. I think the requirements are:
a) logs from std out go to app insights and do not go to system logs
b) logs from std err go to app insights and do not go to system logs
c) a handful (last 5 maybe?) of the most recent std err logs are buffered and send to system logs when we detect the process crashed, and these error logs are sanitized before being sent to system logs.

Reply:
done!